scripts/src/main/java/com/mathewsachin/fategrandautomata/scripts/prefs/IPreferences.kt

```diff
@@ -52,4 +52,4 @@ interface IPreferences {
 
 val IPreferences.wantsMediaProjectionToken get() = !useRootForScreenshots
 
-val IPreferences.isNewUI get() = gameServer in listOf(GameServerEnum.Jp, GameServerEnum.En, GameServerEnum.Kr, GameServerEnum.Cn)
\ No newline at end of file
+val IPreferences.isNewUI get() = gameServer in listOf(GameServerEnum.Jp, GameServerEnum.En, GameServerEnum.Kr, GameServerEnum.Cn, GameServerEnum.Tw)
```

Comment:
isNewUi is no longer needed since that condition now always evaluates to `true`. All code lines for `!isNewUi` can be removed now.

Reply:
I've committed the removal of that code. One thing I noticed is that we're not sure if CN displays FGO over notches or not. See CutoutManager line 77.

Does TW need that?

Reply:
> I've committed the removal of that code. One thing I noticed is that we're not sure if CN displays FGO over notches or not. See CutoutManager line 77.
> 
> Does TW need that?

I cannot be sure about notches, since I don't have a phone with a notch to test atm. Is there any other TW player with a notched phone to confirm?

Reply:
FGO TW display black bolder in notch area on my old phone.

Reply:
I've changed the code to be the same for CN and TW. Assume that the notch area is not used by FGO.